### PR TITLE
TSK-925 Collaborators

### PR DESCRIPTION
**Pull request by BykhovDenis**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues
https://front.hc.engineering/workbench/platform/tracker/TSK-925
A list of closed updated issues

### CODE REVIEW

File: models/notification/src/index.ts [hunk 1]
```diff
@@ -155,24 +166,28 @@ export function createModel (builder: Builder): void {
...
-  builder.createDoc(
-    notification.class.NotificationProvider,
-    core.space.Model,
-    {
-      label: notification.string.BrowserNotification,
-      default: true
-    },
-    notification.ids.BrowserNotification
-  )
+  // Temporarily disabled, we should think about it
```
Comment (haiodo) [inline models/notification/src/index.ts:181]: Note to not allow merge without.

File: packages/presentation/src/components/AttributeBarEditor.svelte [hunk 2]
```diff
@@ -38,72 +38,72 @@
...
+  let editor: AnySvelteComponent | undefined
 
   function onChange (value: any) {
     const doc = object as Doc
     if (draft) {
       ;(doc as any)[attributeKey] = value
       dispatch('update', { key, value })
     } else {
-      updateAttribute(client, doc, _class, { key: attributeKey, attr: attribute }, value)
+      updateAttribute(client, doc, doc._class, { key: attributeKey, attr: attribute }, value)
```
Comment (haiodo) [inline packages/presentation/src/components/AttributeBarEditor.svelte:49]: _class could be mixin, do we loose here something?

File: packages/core/src/tx.ts [hunk 3]
```diff
@@ -201,6 +201,20 @@ export interface PushOptions<T extends object> {
...
+ */
+export interface UnsetProperties {
+  [key: string]: any
+}
+
+/**
+ * @public
+ */
+export interface UnsetOptions {
+  $unset?: UnsetProperties
```
Comment (haiodo) [inline packages/core/src/tx.ts:215]: May be it should be array of fields? why we need record?

**Review by haiodo — APPROVED** [review]

File: packages/core/src/tx.ts (hunk 3 above)
Comment (BykhovDenis) [reply packages/core/src/tx.ts:215]: Like mongo)

File: packages/presentation/src/components/AttributeBarEditor.svelte (hunk 2 above)
Comment (BykhovDenis) [reply packages/presentation/src/components/AttributeBarEditor.svelte:49]: It should be class, not mixin.

File: models/notification/src/index.ts (hunk 1 above)
Comment (BykhovDenis) [reply models/notification/src/index.ts:181]: It will be merged without. I'll return it in Inbox PR, because currently it won't work

**Pull request merged**